doc/classes/@GlobalScope.xml

```diff
@@ -1277,7 +1277,7 @@
 		</member>
 		<member name="JavaScript" type="JavaScript" setter="" getter="">
 			The [JavaScript] singleton.
-			[b]Note:[/b] Only implemented on HTML5.
+			[b]Note:[/b] Only implemented in Web platform.
```

Comment:
```suggestion
			[b]Note:[/b] Only implemented on the Web platform.
```
Not sure if "on" or "in" is better but I guess "the" is needed. Since we say "Implemented on Linux/Windows" etc. I guess "on" is the better option?

Reply:
That being said, given the scope of the changes in this PR, such nitpick is maybe best left to future work. It should be easy enough to grep for "Web" and "web" and review the language on all occurrences, if someone is interested.